docs/xxgloballockreservation.md

```diff
@@ -0,0 +1,101 @@
+| [`reservedTransitionThreshold`](#reservedtransitionthreshold) | Advanced tuning parameter. Changes amount of time spent analyzing an object.                    |
+| [`reservedAbsoluteThreshold`  ](#reservedabsolutethreshold  ) | Advanced tuning parameter. Changes amount of time spent analyzing a class for compatibility.    |
+| [`minimumReservedRatio`       ](#minimumreservedratio       ) | Advanced tuning parameter. Changes aggression level for marking a class as highly compatible.   |
+| [`cancelAbsoluteThreshold`    ](#cancelabsolutethreshold    ) | Advanced tuning parameter. Changes amount of time spent analyzing a class for incompatibility.  |
+| [`minimumLearningRatio`       ](#minimumlearningratio       ) | Advanced tuning parameter. Changes aggression level for marking a class as highly incompatible. |
+
+
+### reservedTransitionThreshold
+        -XX:+GlobalLockReservation:reservedTransitionThreshold=<value>
+
+: | Setting       | Value          | Default               |
+  |---------------|----------------|-----------------------|
+  | `<value>`     | number         | 1                     |
+
+: Number of times an object is locked by the same thread before it is considered reserved minus 2. So, with the default value of 1 and object can be reserved the third time it is locked. Value can be 0-3 inclusive. Values of 4 or higher are treated as infinity.
+
+### reservedAbsoluteThreshold
+        -XX:+GlobalLockReservation:reservedAbsoluteThreshold=<value>
+
+: | Setting       | Value          | Default               |
+  |---------------|----------------|-----------------------|
+  | `<value>`     | number         | 10                    |
+
+: Minimum number of objects of a class that get reserved before the class can be considered highly compatible and objects of that class will get reserved the first time they are locked. Values of 65536 or higher are treated as infinity.
+
+### minimumReservedRatio
+        -XX:+GlobalLockReservation:minimumReservedRatio=<value>
+
+: | Setting       | Value          | Default               |
+  |---------------|----------------|-----------------------|
+  | `<value>`     | number         | 1024                  |
+
+: Minimum ratio of reserved objects to flat objects before a class can be considered highly compatible. Values of 65536 or higher are treated as infinity.
+
+### cancelAbsoluteThreshold
+        -XX:+GlobalLockReservation:cancelAbsoluteThreshold=<value>
+
+: | Setting       | Value          | Default               |
+  |---------------|----------------|-----------------------|
+  | `<value>`     | number         | 10                    |
+
+: Minimum number of objects of a class that get converted to flat before the class can be considered highly incompatible and objects of that class will never be reserved. Values of 65536 or higher are treated as infinity.
```

Comment:
Similar comment to last one. Please can we break the long sentence into 2, and remove the future tense "Objects of that class are never reserved."

Reply:
I'll update this for both the `cancelAbsoluteThreshold` option and `reservedAbsoluteThreshold` option.